lang/cs/Org.Apache.REEF.Wake/packages.config

```diff
@@ -18,6 +18,10 @@ specific language governing permissions and limitations
 under the License.
 -->
 <packages>
+  <package id="Microsoft.Avro.Core" version="0.1.0" targetFramework="net451" developmentDependency="true" />
+  <package id="Microsoft.Avro.Tools" version="0.1.0" targetFramework="net451" developmentDependency="true" />
+  <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net451" developmentDependency="true" />
```

Comment:
If you scroll to the right you will see they are a development dependency and we do not link them as they are a later veriosn. When we switch to the .NET Core / Standard project files they will become a runtime dependency and we can use version numbers defined in the build.prop file.